config.txt
arm_64bit=1
arm_boost=1

dtparam=pciex1
dtparam=pciex1_gen=3
dtoverlay=dwc2,dr_mode=peripheral

# Automatically load overlays for detected cameras
camera_auto_detect=0

# Automatically load overlays for detected DSI displays
display_auto_detect=1

# Automatically load initramfs files, if found
auto_initramfs=1

# Enable DRM VC4 V3D driver
dtoverlay=vc4-kms-v3d
max_framebuffers=2

# Don't have the firmware create an initial video= setting in cmdline.txt.
# Use the kernel's default instead.
disable_fw_kms_setup=1


containers/nextcloud/current-config.txt
Some or all of the below information will be requested if it isn't supplied; for fastest response please provide as much as you can. :heart: 

* Nextcloud Server version:  
  - `Nextcloud 32.0.1`  
* Operating system and version:  
  - `#1 SMP PREEMPT Debian 1:6.12.47-1+rpt1~bookworm (2025-09-16)`  
* Web server and version:  
  - `Server version: Apache/2.4.65 (Debian)`  
* Reverse proxy and version:  
  - `nginx version: openresty/1.27.1.2`  
* PHP version:  
  - `PHP 8.3.27 (cli) (built: Nov 18 2025 02:42:46) (NTS)`
* Is this the first time you've seen this error? (Yes / No):
  - `yes`
* When did this problem seem to first start?
  - `prob direct after installation`
* Installation method _(e.g. AlO, NCP, Bare Metal/Archive, etc.)_
  - `OMV Container`
* Are you using CloudfIare, mod_security, or similar? _(Yes / No)_
  - `nope`

### Configuration
#### Nextcloud
```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "version": "32.0.1.2",
        "loglevel": 3,
        "log_rotate_size": 262144,
        "maintenance": false,
        "maintenance_window_start": 3,
        "default_locale": "DE_CH",
        "default_phone_region": "CH",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "password": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "upgrade.disable-web": true,
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "mysql.utf8mb4": true,
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "htaccess.RewriteBase": "\/",
        "trusted_domains": [
            "srv-nas-pi.kingma:8080",
            "office.kingma.ch",
            "office.king.ma"
        ],
        "overwrite.cli.url": "https:\/\/office.kingma.ch",
        "overwriteprotocol": "https",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smpttimeout": 30,
        "installed": true,
        "mail_smtpauth": true,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "ssl",
        "app_install_overwrite": []
    }
}
```

#### Apps
> Enabled:
>   - activity: 5.0.0-dev.0
>   - admin_audit: 1.22.0
>   - bookmarks: 16.0.1
>   - bruteforcesettings: 5.0.0-dev.0
>   - calendar: 6.1.0
>   - circles: 32.0.0
>   - cloud_federation_api: 1.16.0
>   - comments: 1.22.0
>   - contacts: 8.1.0
>   - contactsinteraction: 1.13.1
>   - cookbook: 0.11.5
>   - dashboard: 7.12.0
>   - dav: 1.34.2
>   - federatedfilesharing: 1.22.0
>   - federation: 1.22.0
>   - files: 2.4.0
>   - files_downloadlimit: 5.0.0-dev.0
>   - files_external: 1.24.0
>   - files_pdfviewer: 5.0.0-dev.0
>   - files_reminders: 1.5.0
>   - files_sharing: 1.24.0
>   - files_trashbin: 1.22.0
>   - files_versions: 1.25.0
>   - firstrunwizard: 5.0.0-dev.0
>   - groupfolders: 20.1.4
>   - logreader: 5.0.0-dev.0
>   - lookup_server_connector: 1.20.0
>   - nextcloud_announcements: 4.0.0-dev.0
>   - notes: 4.12.4
>   - notifications: 5.0.0-dev.0
>   - oauth2: 1.20.0
>   - password_policy: 4.0.0-dev.0
>   - photos: 5.0.0-dev.1
>   - privacy: 4.0.0-dev.0
>   - profile: 1.1.0
>   - provisioning_api: 1.22.0
>   - recommendations: 5.0.0-dev.0
>   - related_resources: 3.0.0-dev.0
>   - richdocuments: 9.0.1
>   - serverinfo: 4.0.0-dev.0
>   - settings: 1.15.1
>   - sharebymail: 1.22.0
>   - support: 4.0.0-dev.0
>   - survey_client: 4.0.0-dev.0
>   - suspicious_login: 10.0.0-dev.0
>   - systemtags: 1.22.0
>   - text: 6.0.1
>   - theming: 2.7.0
>   - twofactor_backupcodes: 1.21.0
>   - twofactor_nextcloud_notification: 6.0.0-dev.0
>   - twofactor_totp: 14.0.0
>   - updatenotification: 1.22.0
>   - user_status: 1.12.0
>   - viewer: 5.0.0-dev.0
>   - weather_status: 1.12.0
>   - webhook_listeners: 1.3.0
>   - workflowengine: 2.14.0
> Disabled:
>   - app_api: 32.0.0 (installed 32.0.0)
>   - documentserver_community: 0.2.1 (installed 0.2.1)
>   - encryption: 2.20.0 (installed 2.19.0)
>   - fulltextsearch: 32.0.0 (installed 32.0.0)
>   - user_ldap: 1.23.0 (installed 1.22.0)


### Summary of the issue you are facing:

[...]

### Steps to replicate it (hint: details matter!):

1. 
2. 
3. 

### Log entries

#### Nextcloud
Please provide the log entries from your Nextcloud log that are generated during the time of problem (via the *Copy raw* option from *Administration settings->Logging* screen or from your `nextcloud.log` located in your data directory). Feel free to use a pastebin/gist service if necessary.
```
PASTE HERE
```



containers/nextcloud/config.php
<?php
// This file is part of Nextcloud.
$CONFIG = [ 
  'htaccess.RewriteBase'     => '/',
  'memcache.local'           => '\\OC\\Memcache\\APCu',
  'apps_paths'               =>
    [ 
      0 =>
        [ 
          'path'     => '/var/www/html/apps',
          'url'      => '/apps',
          'writable' => false,
        ],
      1 =>
        [ 
          'path'     => '/var/www/html/custom_apps',
          'url'      => '/custom_apps',
          'writable' => true,
        ],
    ],
  'memcache.distributed'     => '\\OC\\Memcache\\Redis',
  'memcache.locking'         => '\\OC\\Memcache\\Redis',
  'redis'                    =>
    [ 
      'host'     => 'redis',
      'password' => '',
      'port'     => 6379,
    ],
  'upgrade.disable-web'      => true,
  'instanceid'               => 'aaaaa',
  'passwordsalt'             => 'aaaaa',
  'secret'                   => 'aaaaa',
  'trusted_domains'          =>
    [ 
      0 => 'srv-nas-pi.kingma:8080',
      1 => 'office.kingma.ch',
      2 => 'office.king.ma',
    ],
  'overwrite.cli.url'        => 'http://srv-nas-pi.kingma:8080',
  'overwriteprotocol'        => 'https',
  'datadirectory'            => '/var/www/html/data',
  'dbtype'                   => 'mysql',
  'version'                  => '31.0.4.1',
  'dbname'                   => 'nextcloud',
  'dbhost'                   => 'nextclouddb',
  'dbport'                   => '',
  'dbtableprefix'            => 'oc_',
  'mysql.utf8mb4'            => true,
  'dbuser'                   => 'nextcloud',
  'dbpassword'               => 'dbpassword',
  'installed'                => true,
  'mail_smtpmode'            => 'smtp',
  'mail_sendmailmode'        => 'smtp',
  'mail_from_address'        => 'theking2',
  'mail_domain'              => 'king.ma',
  'mail_smtphost'            => 'cicero.metanet.ch',
  'mail_smtpport'            => '465',
  'mail_smpttimeout'         => 30,
  'mail_smtpsecure'          => 'ssl',
  'loglevel'                 => 2,
  'maintenance'              => false,
  'maintenance_window_start' => 3,
];
